From baeef026f80e19a634a4096e97286419aca1cf68 Mon Sep 17 00:00:00 2001
From: He Zhe <zhe.he@windriver.com>
Date: Thu, 23 Aug 2018 00:22:04 +0800
Subject: [PATCH] fs/ftest/ftest06.c: Fix too small name string and related
 failure

The name string is too small to contain normal full path names and causes
the following failure.

"ftest06     2  TFAIL  :  ftest06.c:223: Can't chdir(): errno=ENOENT(2): No such file or directory"

Upstream-Status: Submitted [ltp-list@lists.sourceforge.net]

Signed-off-by: He Zhe <zhe.he@windriver.com>
---
 testcases/kernel/fs/ftest/ftest06.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testcases/kernel/fs/ftest/ftest06.c b/testcases/kernel/fs/ftest/ftest06.c
index 2ca7c88..b434067 100644
--- a/testcases/kernel/fs/ftest/ftest06.c
+++ b/testcases/kernel/fs/ftest/ftest06.c
@@ -100,7 +100,7 @@ static char *fstyp;
 int main(int ac, char *av[])
 {
 	int pid, child, status, count, k, j;
-	char name[3];
+	char name[MAXPATHLEN];
 
 	int lc;
 
-- 
2.7.4

